main.css

```diff
@@ -0,0 +1,148 @@
+
+.head-triangle {
+    position: relative;
+    top: 10px;
+    left: 10px;
+    width: 0;
+    height: 0;
+    border-left: 300px solid transparent;
+    border-right: 300px solid transparent;
+    border-bottom: 175px solid black;
+}
+
+.white-triangle {
+    position: absolute;
+    top: 20px;
+    left: -300px;
```

Comment:
Negative positioning is something to be avoided, as when you make your css reusable it will become difficult to manage.

I suggest adding a container around the house (`<main class="house">`, maybe!), with `position: relative`. Then you can place your elements inside it without needing negative margins.